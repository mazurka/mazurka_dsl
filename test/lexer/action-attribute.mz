-main.
- test_is_another_test.
-other testing1.
-      attribute_with value.
- funcall users:list(Users).