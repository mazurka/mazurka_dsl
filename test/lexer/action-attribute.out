{attr_begin,1}.
{atom,1,main}.
{def_end,1}.
{attr_begin,2}.
{atom,2,test_is_another_test}.
{def_end,2}.
{attr_begin,3}.
{atom,3,other}.
{atom,3,testing1}.
{def_end,3}.
{attr_begin,4}.
{atom,4,attribute_with}.
{atom,4,value}.
{def_end,4}.
{attr_begin,5}.
{atom,5,funcall}.
{call_begin,5,{users,list}}.
{variable,5,'Users'}.
{call_or_affordance_end,5}.
{def_end,5}.